doc/src/vpr/command_line_usage.rst

```diff
@@ -95,6 +95,14 @@ VPR runs all stages of (pack, place, route, and analysis) if none of :option:`--
 
     **Default:** ``off``
 
+.. option:: --legalize
+
+    Reconstruct a clustering and placement solution from an input flat placement file.
+    Must be used with :option:`--flat_place_file <file>'.
+    Use of this option supercedes option::`--pack`.
```

Comment:
This isn't rendering correctly; maybe because the second option:: should be :option: ?
![image](https://github.com/verilog-to-routing/vtr-verilog-to-routing/assets/5912487/2bfcb646-cdf9-4ec5-8dab-d1f82c901a6f)

Also, the --flat_place_file no longer has to be specified as you've added a default for it, so you can remove that line.

Reply:
Fixed, marking as resolved.